src/popup/utils/constants.ts

```diff
@@ -22,6 +22,22 @@ export const MOBILE_WIDTH = 480;
 
 export const LOCAL_STORAGE_PREFIX = 'sh-wallet';
 
+export const PROTOCOL_AETERNITY = 'aeternity';
+export const PROTOCOL_BITCOIN = 'bitcoin';
+
+export const PROTOCOLS = [
+  PROTOCOL_AETERNITY,
+  PROTOCOL_BITCOIN,
+] as const;
+
+export const PROTOCOL_VIEW_ACCOUNT_DETAILS = 'AccountDetails';
+export const PROTOCOL_VIEW_TRANSFER_RECEIVE = 'TransferReceiveModal';
+
+export const PROTOCOL_VIEWS = [
+  PROTOCOL_VIEW_ACCOUNT_DETAILS,
+  PROTOCOL_VIEW_TRANSFER_RECEIVE,
+] as const;
+
```

Comment:
Better naming may be needed here. eg: `DISTINCT_PROTOCOL_VIEWS` to indicate these are separate views for each protocol